src/config/index.js

```diff
@@ -21,12 +22,13 @@ if (localStorage['defaultNode'] && validURL(localStorage['defaultNode'])) {
 export const Endpoint = {
   api: defaultNode,
   getCurrentNode: () => defaultNode.replace(/(^\w+:|^)\/\//, '').replace(':' + PORT, ''),
-  ws: 'ws://47.107.245.217:' + PORT,
+  ws: `ws://${DEFAULT_HOST}:${PORT}`,
   nodes: [
+    { protocol: 'http', domain: 'api-01.mt.us-west-2.nemtech.network', port: ':' + PORT },
```

Comment:
Maybe it is better to change `domain` to `host`, so it will has correlation with `DEFAULT_HOST`

Reply:
good point.